types/src/connectors/api_handlers/update_connector.ts

```diff
@@ -1,16 +1,7 @@
 import * as t from "io-ts";
 
-import { CreateConnectorUrlRequestBodySchema } from "../../connectors/api_handlers/create_connector";
-
-export const UpdateConnectorOAuthRequestBodySchema = t.type({
-  connectionId: t.union([t.string, t.null, t.undefined]),
-});
-
 export const UpdateConnectorRequestBodySchema = t.type({
-  connectorParams: t.union([
-    CreateConnectorUrlRequestBodySchema,
-    UpdateConnectorOAuthRequestBodySchema,
-  ]),
+  connectionId: t.union([t.string, t.null, t.undefined]),
```

Comment:
Why allow undefined / null here?

Reply:
I think we want to enforce that update pass a connection as in practice this update really is an update connection only?

Reply:
And connectorsAPI enforces it right?